jasa/paper-jasa.tex

```diff
@@ -47,7 +47,7 @@
 \if1\blind
 {
   \title{\bf Approximating Likelihood Ratios with Calibrated Discriminative Classifiers}
-  \author{Kyle Cranmer and Gilles Louppe\\
+  \author{Kyle Cranmer, Juan Pavez and Gilles Louppe\\

```

Comment:
Ping @jgpavez

Given the work with all did together, we added you as author. What shall be your affiliation? 

Also, it would be nice if you could also proofread the current version of the paper :)